### PR TITLE
Support for Concurrent maintenance in IPS Everest

### PR DESCRIPTION
**Pull request by SteffiAntony**
Modified the condition check for displaying the IPS equivalent Everest model along with Everest Concurrent Maintenance page, if it is an Everest model. 
Jira: https://jsw.ibm.com/browse/PFEBMC-2685
Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=614706
Signed-off-by: Steffi Antony <“steffiantony196@gmail.com”>

### CODE REVIEW

**Review by Renuka9527 — APPROVED** [review]

**Comment by Nikhil-Ashoka** [comment]
@SteffiAntony, We need a defect and must-fix approval to get this merged.

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]
`if (this.systemInfo?.startsWith('9043' || '8860'))`
This || will not work.
you should use: 
`if (this.systemInfo?.startWith('9043') || this.systemInfo?.startWith('8860'))`

File: src/env/components/AppNavigation/ibm.js [hunk 1]
```diff
@@ -277,7 +277,10 @@ const AppNavigationMixin = {
       return this.$store?.getters?.['global/currentUser']?.RoleId;
     },
     model() {
-      if (this.systemInfo?.startsWith('9043')) {
+      if (
+        this.systemInfo?.startWith('9043') ||
```
Comment (Nikhil-Ashoka) [inline src/env/components/AppNavigation/ibm.js]: This should be startsWith, After making change, please test with an everest machine to see if it is working as expected

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]
Change startWith to startsWith

File: src/env/components/AppNavigation/ibm.js (hunk 1 above)
Comment (SteffiAntony) [reply src/env/components/AppNavigation/ibm.js]: Resolved

**Review by nabilpandiyalayil — APPROVED** [review]

**Review by Nikhil-Ashoka — APPROVED** [review]

**Pull request merged**